akka-cluster-sharding/src/main/scala/akka/cluster/sharding/ShardCoordinator.scala

```diff
@@ -329,6 +329,8 @@ object ShardCoordinator {
         rememberEntities: Boolean = false)
         extends ClusterShardingSerializable {
 
+      override def toString = s"State($shards)"
```

Comment:
Or more to convention?
```suggestion
      override def toString = s"State[$shards]"
```

Reply:
This is 'case class style' ;) - indeed in the log messages I put `[]`around it as well :+1: